src/v2/Apps/Order/Components/PriceOptions.tsx

```diff
@@ -1,6 +1,7 @@
 import { BorderedRadio, Flex, RadioGroup, Text } from "@artsy/palette"
 import { useEffect, useState } from "react"
 import { OfferInput } from "v2/Apps/Order/Components/OfferInput"
+import { MinPriceWarning } from "./MinPriceWarning"
```

Comment:
Moved the min price notice to its own component since this one was getting chonky